src/components/Footer/Footer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './footer.css';
+
+const Footer = () => (
+  <div className="footer">
```

Comment:
FYI there is `footer` from HTML5 tag used for semantic and not only

Reply:
Updated Css name to avoide using semantic Tag